### ISSUE TITLE
Write test cases for mama project

### ISSUE THREAD

**Issue by breslavsky** [breslavsky/hello-cypress#14]
User as a base https://github.com/breslavsky/hello-cypress/tree/main/test_cases

**Comment by EmelyanovaQA** [comment]
@breslavsky I've corrected check-list. If there are no remarks, I'll finish it

**Comment by breslavsky** [comment]
@EmelyanovaQA please send a link to file or commit

**Comment by EmelyanovaQA** [comment]
@breslavsky https://github.com/breslavsky/hello-cypress/blob/EmelyanovaQA/test_cases/check-list.md

**Comment by breslavsky** [comment]
It looks much better. Let me just suggest format:

# 1. Layout

## 1.1. Header

- [ ] **1.1.1.** Header is visible.
- [ ] **1.1.2.** Logo is presented with link to main page.
- [ ] **1.1.3.** Main menu is presented.

### For anonymous user
- [ ] **1.1.4.** Main menu contains items: Home, Sign Up, Sing Up.

### For logged in user
- [ ] **1.1.5.** Main menu contains items: Home, New Article, Setting, link to user profile page.

## 1.2. Jumbo

### For anonymous user
- [ ] **1.2.1.** Jumbo is visible.
- [ ] **1.1.2.** Jumbo сontains app name and slogan.

### For logged in user
- [ ] **1.1.3.** Jumbo is hidden.

# 2. Features

- [ ] 2.1. User can register.
- [ ] 2.2. User can login.
- [ ] 2.3. User can publish article.

# 3. UI / UX
- [ ] **3.1.** All the mandatory fields should be validated.

# 4. Security
- [ ] **4.1.** HTTPs is used.

I found good article https://www.guru99.com/complete-web-application-testing-checklist.html

**Comment by EmelyanovaQA** [comment]
@breslavsky Check please a new format and I`ll finish in this manner https://github.com/breslavsky/hello-cypress/blob/EmelyanovaQA/test_cases/check-list.md

**Comment by EmelyanovaQA** [comment]
@breslavsky Check, please, check list https://github.com/breslavsky/hello-cypress/blob/EmelyanovaQA/test_cases/check-list.md. While writing it I've noticed that there is no subscription list where registered users could see their subscriptions

**Comment by EmelyanovaQA** [comment]
@breslavsky Are there any remarks?

**Comment by breslavsky** [comment]
It was discussed in Zoom.

**Comment by EmelyanovaQA** [comment]
@breslavsky we didn't discuss if there should be a function to edit a comment and if it should be mentionув in checklist.

**Comment by EmelyanovaQA** [comment]
@breslavsky  yes, but in this app there is no function to edit comments and I wanted to know if the app should have that function because this function is logically necessary and other apps like this one have such function, where registered users write there comments and can edit them.

**Comment by breslavsky** [comment]
Perhaps Conduit has no this feature.

**Comment by EmelyanovaQA** [comment]
I added to check list Contents and corrected some items https://github.com/breslavsky/hello-cypress/blob/EmelyanovaQA/test_cases/check-list.md#edit-article

**Comment by breslavsky** [comment]
@EmelyanovaQA I am very sorry, I like it! Please send pull request to merge!

**Comment by EmelyanovaQA** [comment]
@breslavsky I pushed pull request, waiting for approval
